src/core/lib/iomgr/resolve_address.h

```diff
@@ -41,7 +41,7 @@
 
 typedef struct {
   char addr[GRPC_MAX_SOCKADDR_SIZE];
-  size_t len;
+  uint32_t len;
```

Comment:
I'm doing change size_t to uint32_t change because it make possible to minimal code change.
Without it, pointer arithmetic lost value on big endian.
Then trying with size_t, it needs to define temporary variable.
Then trying to signed type. It break succesfull comparisions in GPR_ASSERT's.
Maybe need to change this to unsined int? I'm think so 4 bytes type is enought for that purpose.